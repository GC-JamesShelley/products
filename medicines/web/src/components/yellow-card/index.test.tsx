import { shallow } from 'enzyme';
import React from 'react';
import YellowCard from './index';

describe(YellowCard, () => {
  it('should render', () => {
    const component = shallow(<YellowCard />);
    expect(component).toMatchSnapshot();
  });
});
