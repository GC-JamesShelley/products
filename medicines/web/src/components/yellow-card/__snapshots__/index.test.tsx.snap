// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YellowCard should render 1`] = `
<styled.section>
  <header
    aria-label="Yellow card"
    role="img"
  >
    <styled.h2>
      Yellow Card
    </styled.h2>
  </header>
  <div
    className="action-bar"
  >
    <styled.div
      role="img"
    >
      <div>
        <SvgYellowCard />
      </div>
    </styled.div>
    <p>
      Report a side effect with a medicine or medical device
    </p>
    <a
      className="primary-button"
      href="https://yellowcard.mhra.gov.uk/"
      title="Yellow Card"
    >
      Make a report
    </a>
  </div>
</styled.section>
`;
